### PR TITLE
mv config ini to toplevel

### PR DESCRIPTION
**Pull request by wenwenchenbosch**
solve #114

### CODE REVIEW

**Comment by wenwenchenbosch** [comment]
@nayakned you can also review this pr quickly.

File: doc/usage.md [hunk 1]
```diff
@@ -36,7 +37,7 @@ Options:
                                         and also shall not fail connections due
                                         to self-signed certificates.
   --use-keycloak                        Use KeyCloak for permission management
-  --address arg (=localhost)            If provided, W3C-Server shall use 
+  --address arg (=127.0.0.1)            If provided, W3C-Server shall use
```
Comment (SebastianSchildt) [inline doc/usage.md:40]: kuksa-val-server

File: src/main.cpp [hunk 2]
```diff
@@ -207,7 +207,7 @@ int main(int argc, const char *argv[]) {
   program_options::options_description desc{"Options"};
   desc.add_options()
     ("help,h", "Help screen")
-    ("config-file,cfg", program_options::value<boost::filesystem::path>()->default_value(boost::filesystem::path{"config.ini"}),
+    ("config-file,c", program_options::value<boost::filesystem::path>()->default_value(boost::filesystem::path{"config.ini"}),
       "Configuration file with W3C-Server input parameters."
```
Comment (SebastianSchildt) [inline src/main.cpp:211]: kuksa-val-server

File: src/main.cpp [hunk 3]
```diff
@@ -250,6 +250,7 @@ int main(int argc, const char *argv[]) {
   if (variables.count("config-file")) {
     auto configFile = variables["config-file"].as<boost::filesystem::path>();
     auto configFilePath = boost::filesystem::path(configFile);
+    std::cout << "Read configs from " <<  configFile.string() <<std::endl;
```
Comment (SebastianSchildt) [inline src/main.cpp:253]: Good!

**Review by SebastianSchildt — COMMENTED** [review]
Looks good. Still want to give it a small test run...

**Review by SebastianSchildt — APPROVED** [review]

**Comment by nayakned** [comment]
Could build the kuksa.val server successfully. Tested with the vss-testclient. Could be merged in my opinion.

**Review by SebastianSchildt — APPROVED** [review]

**Pull request merged**